### ISSUE TITLE
LayerList layers prop behavior

### ISSUE THREAD

**Issue by brambow** [boundlessgeo/sdk#734]
Say I have two layers: an OSM basemap, and a geojson point layer.  I've given each of these layers a metadata property called 'isBasemap', and set it to true for the OSM layer and false for the geojson.

I'm trying to exclude the basemap from the layer list component by passing an array of objects with the following shape:

`[{id: 'layer_id}]`

and only populating that array with layers where isBasemap is false.

However, passing that array to the LayerList component as the layers prop doesn't seem to work, as I'm still seeing the OSM layer in the layer list.

I believe the issue is because the layers prop in the LayerList component is being mapped to the Redux store: state.map.layers.  Unless I'm missing something, this prevents using a custom list of layers to show in the LayerList component.

**Comment by bartvde** [comment]
Good point, we recently introduced a metadata prop for hiding layers from the layerlist, ```bnd:hide-layerlist``` so in your case you could set those to true for the layers you do not want in the layerlist.
```
id: 'my_layer_id',
metadata: {
  'bnd:hide-layerlist': true
}
```

would that work for your use case?
I don't think it is already in a release though.

**Comment by bartvde** [comment]
thanks for the feedback, that is good to hear

**Comment by bartvde** [comment]
@brambow we just released 2.2.0 now https://github.com/boundlessgeo/sdk/releases/tag/v2.2.0 please check it out at your convenience and let us know if the new metadata works for you

**Comment by brambow** [comment]
It works.  Thanks!
